app/src/main/res/values/attrs.xml

```diff
@@ -17,6 +17,10 @@
   -->
 <resources>
 
+    <declare-styleable name="CommonAttributes">
```

Comment:
Is this really used?

Reply:
Attributes with the same name can't be declared twice in the same attrs.xml file, so here I need to declare it once and reference it at the two places that need it